src/sw.test.js

```diff
@@ -58,24 +50,39 @@ describe('Service Worker', () => {
         expect(fetchSpy).toHaveBeenCalledWith(expectedUrl, { mode: 'no-cors' });
       });
     });
+    describe('image requested when webp supported', () => {
+      // The service worker doesn't do anything when webp is supported
+      it.each`
+        image
+        ${`${TEST_IMAGE_URL}/news/puppies.jpg.webp`}
```

Comment:
Could we also add a couple more scenarios here so that we have coverage of all scenarios (look at the test above) e.g. urls with `/ace/standard` etc - we may also want to include a column for the `reason` i.e. why the image is not requested